### ISSUE TITLE
Warning when getting freemarker template

### ISSUE THREAD

**Issue by kirestoj01** [gravitee-io/issues#7037]
During startup we are seeing these warning messages in the logs:
```
WARN  i.g.r.a.s.i.NotificationTemplateServiceImpl - Problem while getting freemarker template PORTAL.USER_REGISTRATION_REQUEST from file : /opt/graviteeio-full-3.15.1/graviteeio-apim-rest-api-3.15.1/templates/notifications/portal/PORTAL.USER_REGISTRATION_REQUEST.yml (No such file or directory)
```

This is one of many warnings for missing templates.
The folder is checked and empty. 
In older releases the flies are available.

Used version 3.15.0, 3.15.1

**Comment by brasseld** [comment]
Hey @kirestoj01 how are you running gravitee ?

**Comment by kirestoj01** [comment]
@brasseld it was on both local and docker installation.

**Comment by phiz71** [comment]
Hi, it's the same root cause as https://github.com/gravitee-io/issues/issues/7057
It has been fixed in 3.15.3

**Comment by ytvnr** [comment]
so can we close it ?

**Comment by gaetanmaisse** [comment]
Closing as it has been fixed with https://github.com/gravitee-io/issues/issues/7057, released in 3.15.3
